content/docs/user-guide/how-to/resolve-merge-conflicts.md

```diff
@@ -80,11 +80,11 @@ versions, then you can follow this process:
 3. Merge it by-hand;
 4. Finally, run `dvc add data.xml` to overwrite the conflicted `.dvc` file.
 
-### Append-only directories
+### Directories
 
-If you have an "append-only" dataset (where people only add new
-files/directories) DVC provides a [Git merge driver] that can automatically
-resolve merge conflicts for you. To use it, first set it up in your Git repo:
+If you have a directory, DVC provides a [Git merge driver] that can
+automatically resolve many merge conflicts for you. To use it, first set it up
```

Comment:
`many` makes me guess 😅 . Is there a short way of clarifying which conflicts can solve an which can't? Or are there just too many possible scenarios to list?

Reply:
Good point. PTAL at the updated text.

Reply:
Let's move the warning to the end of the section, I think.